Docs.md

```diff
@@ -7,6 +7,18 @@ setting up a SugarCRM's REST API testing environment and interacting with it.
 
 You should be familiar with [mocha][mocha] and [JavaScript Promises][google-js-promises].
 
+## Installation and Running
+
+Thorn tests are run using the Mocha test runner. To add Thorn as a dependency, run
+
+```bash
+$ yarn add --dev '@sugarcrm/thorn'
+```
+
+You will need to invoke Mocha in order to run Thorn. Check out
+[Mocha's usage information][mocha-usage], [grunt-mocha], and
+[gulp-spawn-mocha] for information about instantiating Mocha.
```

Comment:
I'm not providing Sugar-specific information here, just a general comment about how to invoke Thorn tests.